pkg/testutils/testutils.go

```diff
@@ -19,5 +19,9 @@ func BuildTestRESTMapper() meta.RESTMapper {
 		schema.GroupVersionKind{Kind: "ConfigMap", Version: "v1"},
 		meta.RESTScopeNamespace,
 	)
+	restMapper.Add(
+		schema.GroupVersionKind{Kind: "Deployment", Version: "v1", Group: "apps"},
```

Comment:
should we have one for a deploymentConfig as well?

Reply:
It is not required in this PR. But I think it's fine to add that.

Reply:
In this particular case, I think it is fine to omit `DeploymentConfig` until we add tests covering its usage.

What do you think?